dev/nginx.minimal.conf

```diff
@@ -0,0 +1,61 @@
+# Main server configuration. See below for redirects.
+server {
```

Comment:
would it be easier to have the server not have https enabled?  that would remove the bulk of the odd bits and remove the need for the separate entrypoint script

Reply:
Ooh yes, good idea. I have a feeling this whole script could be way more minimal, I just copied the main one over and delete the extra routes, but actually I could strip it right back.